### PR TITLE
400560: PG Migration Docker image

### PR DESCRIPTION
**Pull request by ppotturi**
[AB#400560](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/400560)

https://dev.azure.com/defragovuk/DEFRA-FFC/_build/results?buildId=602362&view=results

### CODE REVIEW

File: .azuredevops/publish-docker-image.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  - name: publishFromFeature
+    displayName: Publish from feature branch
+    type: boolean
+    default: false
+
+variables:
+  - name: dockerImageName
+    value: "adp-postgres-migration"
+  - name: dockerImageVersion
+    value: "0.0.1"
```
Comment (rajeshkaremane) [inline .azuredevops/publish-docker-image.yaml]: start version should be 0.1.0?

File: README.md [hunk 2]
```diff
@@ -3,13 +3,39 @@
```
Comment (rajeshkaremane) [inline README.md:3]: Please add .gitignore and pr template file

Comment (ppotturi) [reply README.md:3]: There are no files to ignore

File: .azuredevops/publish-docker-image.yaml (hunk 1 above)
Comment (DaRoszko) [reply .azuredevops/publish-docker-image.yaml]: agreed, or 1.0.0 if the first main relase

**Review by rajeshkaremane — DISMISSED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    apt-get clean && \
+    apt-get autoremove -y && \
+    rm -rf /var/lib/apt/lists/* /packages-microsoft-prod.deb
+
+RUN pwsh -Command "Set-PSRepository -Name PSGallery -InstallationPolicy Trusted"
+RUN pwsh -Command "Install-Module -Name Az.Accounts -Force -AllowClobber"
+RUN pwsh -Command "Install-Module -Name Az.KeyVault -Force -AllowClobber"
+
+WORKDIR /
+
```
Comment (jeevankuduvaravindran) [inline Dockerfile:72]: @ppotturi Can we change the user back to liquibase to run the db migration via powershell if root user isnt required after installing modules?

**Review by rajeshkaremane — APPROVED** [review]

**Review by kbabigumira — APPROVED** [review]

**Pull request merged**